pages/tutorial/simulations.md

```diff
@@ -189,9 +189,9 @@ More about the environments shipped with the distribution [here][Environments].
 
 #### The `positions` key
 
-The `position` section list the coordinate types of the simulation. Actually, only one value is taken into account. Each type implements the interface [Position][Position]. If no fully qualified position name is provided for class loading, Alchemist uses the package [positions][PositionPackage] to search for the class.
+The `positions` section lists the coordinate types of the simulation. In reality, only one value is taken into account. Each type implements the interface [Position][Position]. If no fully qualified position name is provided for class loading, Alchemist uses the package [positions][PositionPackage] to search for the class.
```

Comment:
Here actually works just fine.

Reply:
"Actually" and "In reality" have basically the same meaning, but from my experience "actually" is usually used when speaking since it carries a meaning of "the previous statement isn't entirely true", as if to correct some part of the previous statement (which doesn't make this usage invalid though).
Because of this, while it is not wrong to use it here, to me it felt out of place compared to the rest of the text so I considered switching it with a more formal expression.

Reply:
I believe such distinction does not hold:
```
actually
ˈaktʃʊəli
adverb
1. as the truth or facts of a situation.
2. used to emphasize that something someone has said or done is surprising.
```

Here it is used with meaning 1, and it is much more common than "in reality" in my experience.